packages/core-data/src/entities.js

```diff
@@ -135,7 +135,7 @@ function* loadPostTypeEntities() {
 			mergedEdits: { meta: true },
 			getTitle( record ) {
 				if ( name === 'wp_template_part' || name === 'wp_template' ) {
```

Comment:
A similar question applies, why the same logic can't be applied on the server? It gets further propagated to more places.

Reply:
I guess with the `useTemplateInfo` hook this might not be needed at all. 🤔 
Will give this a try as well.